src/System.Xml.XmlSerializer/src/System/Xml/Serialization/CodeIdentifiers.cs

```diff
@@ -20,20 +20,19 @@ namespace System.Xml.Serialization
     /// </devdoc>
     internal class CodeIdentifiers
     {
-        private Hashtable _identifiers;
-        private Hashtable _reservedIdentifiers;
+        private Dictionary<string, object> _identifiers;
+        private Dictionary<string, object> _reservedIdentifiers;
         private ArrayList _list;
```

Comment:
Interesting, I don't see a dependency on `System.Collections.NonGeneric` in project.json. What's pulling in that dependency?